tasks/request_files/request_files.py

```diff
@@ -144,21 +144,24 @@ def task(
             LOGGER.info(msg)
             retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
     except KeyError:
-        LOGGER.warn(f"Invalid RESTORE_RETRIEVAL_TYPE: 'None' defaulting to 'Standard'")
+        LOGGER.warn("Invalid RESTORE_RETRIEVAL_TYPE: 'None' defaulting to 'Standard'")
         retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
 
     # Get QUEUE URL
     db_queue_url = str(os.environ[OS_ENVIRON_DB_QUEUE_URL_KEY])
 
     # Use the default glacier bucket if none is specified for the collection or otherwise given.
-    event[EVENT_CONFIG_KEY][CONFIG_GLACIER_BUCKET_KEY] = \
-        event[EVENT_CONFIG_KEY].get(CONFIG_COLLECTION_BUCKET_KEY,
-                                    None)
-    if event[EVENT_CONFIG_KEY][CONFIG_GLACIER_BUCKET_KEY] is None:
-        event[EVENT_CONFIG_KEY][CONFIG_GLACIER_BUCKET_KEY] = event[EVENT_CONFIG_KEY].get(
+    try:
```

Comment:
Would it make sense to pull this logic out into a small function for easier testing/replication? Then setting a variable that gets passed throughout?

Reply:
Done

Reply:
Done